lms/djangoapps/learner_home/serializers.py

```diff
@@ -238,11 +288,11 @@ class LearnerEnrollmentSerializer(serializers.Serializer):
     course = CourseSerializer()
     courseRun = CourseRunSerializer(source="*")
     enrollment = EnrollmentSerializer(source="*")
+    certificate = CertificateSerializer(source="*")
 
     # TODO - remove "allow_null" as each of these are implemented, temp for testing.
```

Comment:
Comment calls to remove the `allow_null` -- are these implemented and we can drop?

Reply:
Nope, those still need implementation.